### PR TITLE
Organize Opsgenie provider classes

### PR DESCRIPTION
**Pull request by eladkal**
Organize Opsgenie classes.
~~Since https://github.com/apache/airflow/pull/20263 is a breaking change I want to take the opportunity to organize the classes along with it.~~

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by eladkal** [comment]
Ahh wait. Not ready yet. mark as draft.

File: airflow/providers/opsgenie/CHANGELOG.rst [hunk 1]
```diff
@@ -29,6 +29,16 @@ Removed the ``OpsegnieAlertHook.execute`` method and replaced it with ``Opsegnie
...
 
+Changes were made to classes paths:
+
++-----------------------+-----------------------------------------------------+-----------------------------------------------+
+| Class                 | Old Path                                            | New Path                                      |
++-----------------------+-----------------------------------------------------+-----------------------------------------------+
+| OpsgenieAlertHook     | airflow.providers.opsgenie.hooks.opsgenie_alert     | airflow.providers.opsgenie.hooks.opsgenie     |
++-----------------------+-----------------------------------------------------+-----------------------------------------------+
+| OpsgenieAlertOperator | airflow.providers.opsgenie.operators.opsgenie_alert | airflow.providers.opsgenie.operators.opsgenie |
++-----------------------+-----------------------------------------------------+-----------------------------------------------+
```
Comment (kaxil) [inline airflow/providers/opsgenie/CHANGELOG.rst]:
```suggestion
+---------------------------+---------------------------------------------------------+---------------------------------------------------+
| Class                     | Old Path                                                | New Path                                          |
+---------------------------+---------------------------------------------------------+---------------------------------------------------+
| ``OpsgenieAlertHook``     | ``airflow.providers.opsgenie.hooks.opsgenie_alert``     | ``airflow.providers.opsgenie.hooks.opsgenie``     |
+---------------------------+-------------------------------------------------------------+-----------------------------------------------+
| ``OpsgenieAlertOperator`` | ``airflow.providers.opsgenie.operators.opsgenie_alert`` | ``airflow.providers.opsgenie.operators.opsgenie`` |
+---------------------------+---------------------------------------------------------+---------------------------------------------------+
```

**Review by kaxil — APPROVED** [review]
Minor suggestion

File: airflow/providers/opsgenie/CHANGELOG.rst (hunk 1 above)
Comment (eladkal) [reply airflow/providers/opsgenie/CHANGELOG.rst]: I made the paths backward compatible so no need for the table at the end :)

**Comment by eladkal** [comment]
Should be OK now

**Comment by eladkal** [comment]
transient errors of helm tests. Not related to this change.
merging

**Pull request merged**